.github/workflows/swift-toolchain.yml

```diff
@@ -2348,8 +2348,10 @@ jobs:
       # TODO(issues/154): Preload WixToolset.Sdk on Azure images so we can remove this step.
       - name: Install WixToolset.Sdk
         run: |
-           Register-PackageSource -Name NuGet -Location https://www.nuget.org/api/v2 -ProviderName NuGet
-           Install-Package -Name WixToolset.Sdk -RequiredVersion 4.0.1 -Force
+          if (!(Get-PackageSource | %{ $_.Location } | Select-String -Pattern api.nuget.org)) {
```

Comment:
Should this be using `[string]::IsNullOrEmpty` instead of the `!`? Likewise throughout.

The `% { $_.location }` get objects, and the `Select-String` will give you a string right? I don't know how the boolean conversion is defined in PowerShell.

Reply:
In that it's more clear what's happening? I think the boolean conversion is well defined for empty string: https://stackoverflow.com/a/13739662, but happy to change it to avoid powershell-isms.

Reply:
Done.